rb/Gemfile.lock

```diff
@@ -174,14 +173,9 @@ GEM
     yard (0.9.37)
 
 PLATFORMS
-  java
-  ruby
-  universal-java-1.8
+  arm64-darwin-24
   universal-java-17
-  universal-java-18
-  universal-java-21
-  x64-mingw32
-  x86_64-darwin-22
+  x86_64-linux
```

Comment:
We need to keep all platforms to avoid strange errors on Windows, differents versions of macOS, etc.